src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
+const table = document.querySelector('.field');
+const tbody = table.querySelector('tbody');
+const appendColum = document.querySelector('.append-column');
+const removeColum = document.querySelector('.remove-column');
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+
+appendRow.addEventListener('click', () => {
+  const createRow = document.createElement('tr');
+  const columnCount = table.rows[0].cells.length;
+
+  for (let i = 0; i < columnCount; i++) {
+    const td = document.createElement('td');
+
+    createRow.append(td);
+  }
+  tbody.appendChild(createRow);
+});
+
+removeRow.addEventListener('click', () => {
+  tbody.deleteRow(tbody.rows.length - 1);
```

Comment:
Consider adding a check to ensure that there is at least one row in the table before attempting to delete a row. This will prevent errors when trying to delete a row from an empty table.